src/System.Memory/tests/Performance/Perf.ReadOnlySequence.First.cs

```diff
@@ -0,0 +1,367 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Memory.Tests;
+using System.MemoryTests;
```

Comment:
What helpers/etc are we using from this namespace?

Reply:
We using `BufferSegment` and `CustomMemoryForTest` from usual tests.